### PR TITLE
Refactored code

### PR DESCRIPTION
**Pull request by X1nto**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -62,20 +62,20 @@ module.exports = class Brainfuck extends Plugin {
 			const { body } = await post(`${this.hastebin}documents`).send(text);
 			return `The result was too long, so I uploaded it to hastebin instead!\n\n${this.hastebin}${body.key}`;
 		} catch (err) {
-			console.error(err);
+			this.errorlog(err);;
```
Comment (Vendicated) [inline index.js]: why is there two semis

**Review by Vendicated — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (X1nto) [reply index.js]: oops

**Review by Vendicated — APPROVED** [review]

**Comment by Vendicated** [comment]
superseded by https://github.com/Vendicated/powercord-brainfuck/commit/d1e309ac043970908707796a9836bb96a1dd0bee

**Comment by X1nto** [comment]
btch you could merge it 1 month ago

**Comment by Vendicated** [comment]
HOSE MAD HOSE MAD 🇰🇵

**Pull request closed without merging**